docs/examples/mab-thompson-sampling-tempo/README.ipynb

```diff
@@ -9,17 +9,9 @@
   },
```

Comment:
<p>Would we want to remove all the outputs from this notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/tempo/pull/178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Yeah I feel maybe this notebook needs to be moved to incubating or updated with more docs?

Reply:
Ok makes sense i can update with more docs